addons/payment/views/portal_templates.xml

```diff
@@ -142,18 +142,95 @@
                 <div class="container">
                     <!-- Portal breadcrumb -->
                     <t t-call="payment.portal_breadcrumb"/>
-                    <!-- Status page -->
-                    <div name="o_payment_status" class="row justify-content-center my-3">
-                        <div name="o_payment_status_content"
-                             class="col-12 col-lg-8">
-                            <!-- The content is generated in JavaScript -->
+                    <!-- Payment status page -->
+                    <div class="row justify-content-center my-3">
+                        <div class="col-12 col-lg-8">
+                            <div t-if="payment_not_found" class="text-center">
+                                <p>We're unable to find your payment, but don't worry.</p>
```

Comment:
This has the power to make customers' anxiety skyrocket :smile: 
Let's find a less alarming message; in the end, the transaction will be post-process in the next few minutes anyway.

Reply:
:ok_hand: